src/main/java/com/minecolonies/coremod/util/TeleportToColony.java

```diff
@@ -0,0 +1,85 @@
+package com.minecolonies.coremod.util;
+
+import com.minecolonies.coremod.colony.Colony;
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.colony.IColony;
+import com.minecolonies.coremod.commands.MinecoloniesCommand;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.commands.MinecoloniesCommand'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)